src/hotspot/cpu/riscv/c2_MacroAssembler_riscv.cpp

```diff
@@ -1382,15 +1382,180 @@ void C2_MacroAssembler::string_indexof_linearscan(Register haystack, Register ne
   bind(DONE);
 }
 
+// Compare longwords
+void C2_MacroAssembler::string_compare_long_same_encoding(Register result, Register str1, Register str2,
+                                                  const bool isLL, Register cnt1, Register cnt2,
+                                                  Register tmp1, Register tmp2, Register tmp3,
+                                                  const int STUB_THRESHOLD, Label *DONE, Label *STUB) {
```

Comment:
Personally I prefer to put together STUB_THRESHOLD and STUB, like : `const int STUB_THRESHOLD, Label *STUB, Label *DONE)`. Similar for `C2_MacroAssembler::string_compare_long_different_encoding`.